pkg/alertmanager/multitenant_test.go

```diff
@@ -2163,6 +2167,25 @@ receivers:
 	require.Contains(t, err.Error(), errRateLimited.Error())
 }
 
+func TestMultitenantAlertmanager_computeFallbackConfig(t *testing.T) {
+	// If no fallback config is set, it should return a valid empty config.
+	fallbackConfig, err := ComputeFallbackConfig("")
+	require.NoError(t, err)
+
+	_, err = amconfig.Load(string(fallbackConfig))
+	require.NoError(t, err)
+
+	// If a fallback config file is set, it should return it's content.
```

Comment:
```suggestion
	// If a fallback configuration file is set, it returns its content.
```

Reply:
Spell out terms through for content in plain text. Feel free to ignore this nit if you are pressed for time.